### DISCUSSION TITLE
Provides shared state/fixture data across tests in the different assemblies

### DISCUSSION THREAD

**Discussion by sriramtummala87** [xunit/xunit#2707] [Question] [answered]
Hi,

Is there a way to share data across tests in different assemblies?
We are trying to have multiple test projects for different use cases but want to boostrap the assemblies only once and share it across tests in different assemblies

Thanks in Advance

**Comment by DarylRoberts** [comment]
Yes

**Reply by sriramtummala87** [reply]
Can you please suggest me with those options. All I can see is a collection fixture shared across classes in same assembly. But cannot find a solution for sharing across multiple assemblies

**Comment by bradwilson** [comment, marked as answer]
There is no in-memory object that can be shared between multiple assemblies.

Any data that is shared across multiple assemblies must be shared externally (for example, via the file system).
